.github/workflows/build.yml

```diff
@@ -10,30 +10,36 @@ on:
 jobs:
   build:
     env:
-        NUGET_API_KEY: ${{ secrets.NUGET_API_KEY }}
-        
-    runs-on: ${{ matrix.os }}
+      NUGET_API_KEY: ${{ secrets.NUGET_API_KEY }}
+    runs-on: ubuntu-latest
+    container: debian:oldoldstable
```

Comment:
Would it not be more stable to just fix the version in place so that we manually control which debian/ubuntu version we are compiling against?